spec/support/schemas/letters.json

```diff
@@ -26,62 +26,78 @@
-              "type": "string"
-            },
-            "address_line2": {
-              "type": "string"
-            },
-            "address_line3": {
-              "type": "string"
-            },
-            "city": {
-              "type": "string"
-            },
-            "state": {
-              "type": "string"
-            },
-            "country": {
-              "type": "string"
-            },
-            "foreign_code": {
-              "type": "string"
+            "address": {
+              "description": "User's address in EVSS that will be included in the letter",
+              "properties": {
+                "full_name": {
+                  "type": "string"
+                },
+                "address_line1": {
+                  "type": "string"
+                },
+                "address_line2": {
+                  "type": "string"
+                },
+                "address_line3": {
+                  "type": "string"
+                },
+                "city": {
+                  "type": "string"
+                },
+                "state": {
+                  "type": "string"
+                },
+                "country": {
+                  "type": "string"
+                },
+                "foreign_code": {
+                  "type": "string"
+                },
+                "zip_code": {
+                  "type": "string"
+                }
+              },
+              "type": "object",
+              "required": [
+                "full_name",
+                "address_line1",
+                "address_line2",
+                "address_line3",
+                "city",
+                "state",
+                "country",
+                "foreign_code",
+                "zip_code"
+              ]
             },
-            "zip_code": {
-              "type": "string"
+            "status": {
+              "description": "EVSS API may return an error message in a 200 response, vets-api catches all errors other than not found as server error",
+              "enum": [
+                "OK",
+                "NOT_AUTHORIZED",
+                "NOT_FOUND",
+                "SERVER_ERROR"
+              ]
             }
-          },
-          "type": ["object", "null"],
-          "required": [
-            "full_name",
-            "address_line1",
-            "address_line2",
-            "address_line3",
-            "city",
-            "state",
-            "country",
-            "foreign_code",
-            "zip_code"
-          ]
+          }
         },
-        "status": {
-          "description": "EVSS API may return an error message in a 200 response, vets-api catches all errors other than not found as server error",
-          "enum": [
-            "OK",
-            "NOT_AUTHORIZED",
-            "NOT_FOUND",
-            "SERVER_ERROR"
-          ]
+        {
+          "properties": {
+            "status": {
+              "enum": [
```

Comment:
This enum is reused in multiple places.  Outside the scope of this PR but it probably warrants it's own file with a singular definition.